### PR TITLE
various typos in security.md fr [skip ci]

### PR DESCRIPTION
**Pull request by jaaf**
| Q | A |
| --- | --- |
| Is bugfix? | yes/no |
| New feature? | yes/no |
| Breaks BC? | yes/no |
| Tests pass? | yes/no |
| Fixed issues | comma-separated list of tickets # fixed by the PR, if any |

### CODE REVIEW

File: docs/guide-fr/security-authorization.md [hunk 1]
```diff
@@ -480,7 +480,7 @@ $auth->addChild($admin, $author);
 // ... ajoute les  permissions en tant qu'enfant de  $admin ...
 ```
 
-Notez que dans ce qui est présenté ci-dessus, comme « author » est ajouté en tant qu'enfant de « admin », lorsque vus implémentez la méthode `execute()` de la classe de règle, vous devez respecter cette hiérarchie elle aussi. C'est pourquoi, le nom de rôle est « author », la méthode `execute()` retourne `true` (vrai) si le groupe de l'utilisateur est soit 1, soit 2 (ce qui signifie que l'utilisateur est soit dans le groupe « admin », soit dans le groupe « author »).
+Notez que dans ce qui est présenté ci-dessus, comme « author » est ajouté en tant qu'enfant de « admin », lorsque vous implémentez la méthode `execute()` de la classe de règle, vous devez respecter cette hiérarchie elle aussi. C'est pourquoi,lorsque le nom de rôle est « author », la méthode `execute()` retourne `true` (vrai) si le groupe de l'utilisateur est soit 1, soit 2 (ce qui signifie que l'utilisateur est soit dans le groupe « admin », soit dans le groupe « author »).
```
Comment (SilverFire) [inline docs/guide-fr/security-authorization.md]: Space after `,` is missing

**Comment by jaaf** [comment]
Ok, corrected.  Is this automatic detection ?

**Comment by SilverFire** [comment]
No, it's my manual review :)

**Comment by SilverFire** [comment]
Thank you

**Comment by jaaf** [comment]
Then, thank you! Do you speak French, Dmitry ?

**Comment by SilverFire** [comment]
No, I don't, unfortunately. I speak Ukrainian, Russian and English.

**Pull request merged**